values.yaml

```diff
@@ -417,7 +420,6 @@ redis:
 
   # If set use as redis secret
   passwordSecretName: ~
-  brokerURLSecretName: ~
```

Comment:
I removed this since as best as I can tell, it isn't being used by the platform at all and was only used in a truthy check to prevent the `redis-password` Secret from being created. Using `redis.passwordSecretName` alone is sufficient to do this:

https://github.com/astronomer/airflow-chart/blob/f224a0753fdff15b3063694bf1c69850f67273c9/templates/secrets/redis-secrets.yaml#L5